cli/azd/cmd/up.go

```diff
@@ -52,121 +43,67 @@ func newUpCmd() *cobra.Command {
 }
 
 type upAction struct {
-	flags                      *upFlags
-	env                        *environment.Environment
-	projectConfig              *project.ProjectConfig
-	packageActionInitializer   actions.ActionInitializer[*packageAction]
-	provisionActionInitializer actions.ActionInitializer[*provisionAction]
-	deployActionInitializer    actions.ActionInitializer[*deployAction]
-	console                    input.Console
-	runner                     middleware.MiddlewareContext
-	prompters                  prompt.Prompter
-	provisioningManager        *provisioning.Manager
+	flags                        *upFlags
+	console                      input.Console
+	env                          *environment.Environment
+	projectConfig                *project.ProjectConfig
+	runner                       middleware.MiddlewareContext
+	workflowRunActionInitializer actions.ActionInitializer[*workflowRunAction]
+	provisioningManager          *provisioning.Manager
+}
+
+var defaultUpWorkflow = []*workflow.Step{
+	{Command: "package --all"},
```

Comment:
Why is `--all` not in the `Args` part of this object?

Reply:
I originally created the struct to separate out the command from args but then realized that if we are using cobra to dispatch the command that we don't need to support this separation in the configuration

Reply:
I am slightly nervous about us tying ourselves to having to do argument splitting.  Maybe we should make Command take a `[]string` and just document it as "the args passed to `azd`"?